src/core/components/manager.ts

```diff
@@ -102,6 +105,30 @@ export class StateManager {
 
         _.each(this._topLevelComponents, (component) => component.loadState(this._nextState));
     }
+
+
+    /**
+     * Saves this component's current state and returns it in partially serialized format.
+     * This format retains undefined, NaN, and Infinity values through JSON.stringify.
+     */
+    public saveSerializableState(): any {
```

Comment:
Is the other `save()` ever "publicly" used or did it become an internal functionality? Maybe it should not be public to prevent confusion.

Reply:
If current `save` should indeed not be public, then `saveSerializableState` should be renamed to `save` and rename `save` to something else.

Reply:
I think we should keep both public. I wrote more docs about when to use each